src/api/app/views/webui/request/_show_project_maintainers.html.haml

```diff
@@ -0,0 +1,13 @@
+.modal.fade#show-project-maintainers-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'confirm-modal-label', hidden: true } }
```

Comment:
@ncounter this is the same as the `_show_package_maintainers` partial besides the name `Package` and `Project`. Maybe you can adapt the package one to work for both and use it for both

Reply:
That makes sense, thanks for the pointer. I refactored it in https://github.com/openSUSE/open-build-service/pull/13383/commits/7cd65162f1964377a9fe8478d6a4faab2064a28f , the refactoring includes few changes outside of the beta because the former partial `_show_package_maintainers.html.haml` was used by both of the feature versions.